pages/status.tsx

```diff
@@ -67,13 +73,13 @@ const Status: FC = () => {
       )
   }, [formik, t])
 
-  const handleReset: MouseEventHandler<HTMLButtonElement> = useCallback(
-    (e) => {
-      e.preventDefault()
-      formik.resetForm()
-    },
-    [formik]
-  )
+  // const handleReset: MouseEventHandler<HTMLButtonElement> = useCallback(
```

Comment:
This commented block of code should be removed. We can retreive it in previous commits if needed.